pkg/private/xtest/graphupdater/graph.go

```diff
@@ -136,6 +146,33 @@ func (g *Graph) edges() []string {
 	return res
 }
 
+func (g *Graph) ifIDs() []string {
+	data := make(map[string][]string)
+	for _, l := range g.links {
+		if _, ok := data[l.Src.ia]; !ok {
+			data[l.Src.ia] = make([]string, 0)
+		}
+		if _, ok := data[l.Dst.ia]; !ok {
+			data[l.Dst.ia] = make([]string, 0)
+		}
+		srcIfID := g.IfaceIds[l.Src]
+		dstIfID := g.IfaceIds[l.Dst]
+		src := fmt.Sprintf(`br%s-%d %d%d`,
+			strings.ReplaceAll(l.Src.ia, ":", "_"), len(data[l.Src.ia])+1, srcIfID, dstIfID)
+		dst := fmt.Sprintf(`br%s-%d %d%d`,
+			strings.ReplaceAll(l.Dst.ia, ":", "_"), len(data[l.Dst.ia])+1, dstIfID, srcIfID)
+		data[l.Src.ia] = append(data[l.Src.ia], fmt.Sprintf(`  %s: %s`, src, dst))
+		data[l.Dst.ia] = append(data[l.Dst.ia], fmt.Sprintf(`  %s: %s`, dst, src))
+	}
+	sortedIsds := sortedKeys(data)
+	var res []string
```

Comment:
```suggestion
	res := make([]string, 0, len(sortedIsds)*2)
```

Since res will be appended to at least twice in the loop

Reply:
Done.
<!-- Reviewable comment -OQqvFxSBQgUxQ5CQIFZ:b-896fix -->
<!-- Sent from Reviewable.io -->